snaxc/transforms/snax_copy_to_dma.py

```diff
@@ -485,16 +490,23 @@ def match_and_rewrite(self, op: CopyOp, rewriter: PatternRewriter):
         rewriter.replace_op(op, outermost_for_loop)
 
 
+@dataclass(frozen=True)
 class SNAXCopyToDMA(ModulePass):
     """
     This pass translates memref copies to snitch DMA calls.
     """
```

Comment:
The motivation for this PR seems useful to be part of the documentation of this pass. How about:
```suggestion
    """
    This pass translates memref copies to snitch DMA calls.
    
    If you want to ignore all transformations made by the dma because it makes everything very slow, the    test_ignore_transform gives you the option to replace it by a regular 1D transfer. 
    Note that this renders the data incorrect, yet it can be helpful during debugging
    """
```